apps/acrs/translations/src/en/pages.json

```diff
@@ -68,7 +68,8 @@
     "paragraph": "We may use this email address to contact them for more information about the referral."
   },
   "referrer-email": {
-    "header": "SKELETON: /referrer-email"
+    "header": "Does {{values.full-name}} have an email address?",
```

Comment:
Check Figma for latest text

Reply:
There's more to this than a change of text therefore it can go out in a different pr.